packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -1067,9 +1067,23 @@ export function getValidatorApi(
       };
     },
 
-    // TODO Electra: Implement getAggregatedAttestation to properly handle pre-electra
-    async getAggregatedAttestation() {
-      throw new Error("Not implemented. Use getAggregatedAttestationV2 for now.");
+    async getAggregatedAttestation({attestationDataRoot, slot}) {
```

Comment:
We need to throw an error here if request is for electra+ aggregated attestation as v1 does not support it